index.html

```diff
@@ -629,7 +629,101 @@ <h3>Presentations</h3>
 that are often, but not required to be, related.
         </p>
 
-      </section>
+    </section>
+    <section>
+      <h3>Lifecycle</h3>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+This section seeks to illuminate the lifecycle of a credential. Credentials 
+allow one to make statements about a subject. Moreso, it provides a succint 
+composition mechanism for tamper-evident claims that provide cryptographic 
+guarantees on the provenance of those claims. Presentations enable subjects to 
+use one or more credentials to satisfy requests from a verifier and assert 
+proof of control of those credentials. Credentials, when in motion, form a 
+general flow. This flow provides the bedrock for the lifecycle of a 
+credential, it highlights the issuance of credentials to their verification.
+      </p>
+
+      <p>
+The lifecycle of credentials will take a common path: issuance
+of claims to a credential subject, storage of credentials into credential
```

Comment:
Simplify to: "issuance of one or more credentials, storage of credentials, and then the composition and presentation of credentials to a verifier.